src/components/Common/FormattedDate.svelte

```diff
@@ -9,14 +9,17 @@
 	export let absolute = false;
 
 	$: dateObj = isValidDate(date) ? date : dateFromUnix(date);
-	$: formatFunction = $configStore.scorePreferences.dateFormat == 'full' ? formatDate : formatDateRelative;
-	$: titleFormatFunction = $configStore.scorePreferences.dateFormat == 'full' ? formatDateRelative : formatDate;
-	$: dateTitle = (configStore, $configStore, dateObj && !absolute ? titleFormatFunction(dateObj) : null);
+	$: dateFormat = $configStore.scorePreferences.dateFormat == null ? 'relative' : $configStore.scorePreferences.dateFormat;
+	$: formatFunction = dateFormat == 'full' ? formatDate : 
+						(dateFormat == 'relative' ? formatDateRelative : formatDateCustom);
+	$: titleFormatFunction = dateFormat == 'full' ? formatDateRelative : 
+						     (dateFormat == 'relative' ? formatDate : formatDateCustom);
```

Comment:
This will lead to `custom` having the tooltip identical to the displayed date. Maybe let's also unify this into a single function `formatDateCustomTooltip` and return `full` for `relative` and `relative` for everything else.